docs/en/observability/cloud-monitoring/aws/monitor-amazon-s3.asciidoc

```diff
@@ -0,0 +1,186 @@
+[[monitor-amazon-s3]]
+= Monitor Amazon Simple Storage Service (S3)
+
+++++
+<titleabbrev>Monitor S3</titleabbrev>
+++++
+
+Amazon Simple Storage Service (S3) is an object storage service that offers
```

Comment:
Add generic link to info about the service. Make sure the link is "future proof" (unlikely to change)